modules/core/src/test/scala/gem/LocationSpec.scala

```diff
@@ -70,7 +70,7 @@ class LocationSpec extends FlatSpec with Matchers with PropertyChecks with Arbit
     }
   }
 
-  it should "produce a sorted list of Location.Middle" in {
+  ignore should "produce a sorted list of Location.Middle" in {
```

Comment:
This is failing with a `Maximum call stack size exceeded` error on JS. The stack size is smaller on JS so it produces this error. Any ideas to change `find` to make it stack safe? Otherwise we could make `count` smaller